### PR TITLE
246 presentation logic separation

### PR DESCRIPTION
**Pull request by piggydoughnut**
Separation of Presentation (css) From logic. Imho it is easier to read now.
I also moved all the inline css to separate files too.

I did not edit all the files, only removed big css chunks which were decreasing readability. Some components kept their css mainly coz they are only made up of css. The ones with a bit of logic and structure needed it moved.

### CODE REVIEW

File: src/lib/dev.ts [hunk 1]
```diff
@@ -1,5 +1,3 @@
-import { IMessage } from "../components/Game/onMessage";
```
Comment (piggydoughnut) [inline src/lib/dev.ts:1]: lint changes

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -61,40 +66,11 @@ const Table: React.FunctionComponent = () => {
...
-            `}
-          >
-            <div
-              css={css`
-                position: absolute;
-              `}
-            />
+          <GameTypeWrapper>{gameType}</GameTypeWrapper>
+          <TableWrapper>
+            <Absolute />
```
Comment (piggydoughnut) [inline src/components/Table/Table.tsx:73]: I kept it for no reason :) I think it should go, unless it is some empty div magic I dont understand @norbertdragan :)

File: src/components/Table/Table.tsx [hunk 3]
```diff
@@ -61,40 +66,11 @@ const Table: React.FunctionComponent = () => {
...
-          </div>
-          <div
-            css={css`
-              position: absolute;
-              width: 100%;
-              height: 100%;
-              z-index: 1;
-            `}
-          >
-            <div
```
Comment (piggydoughnut) [inline src/components/Table/Table.tsx:93]: I removed this coz imho empty div does nothing, but correct me if I am wrong :) Is it the same empty div magic as in the file below? @norbertdragan

**Comment by norbertdragan** [comment]
@piggydoughnut Nice work! Not sure what the empty divs are, but if the UI didn't change, it should be fine.

The point of using `styled-components` is so that we can have CSS and JSX in the same file. I agree that the inline styles were making the code difficult to read, so moving them outside the function component is a good call. However, having the styles in `styled-components` and also in separate files is a bit unusual pattern to me (because just use CSS or SASS then).

Also, I think you should look into Storybook. We are using it with Chromatic and it's great at seeing exactly what UI changes are happening when we change the components.

**Comment by piggydoughnut** [comment]
@norbertdragan thank you for checking this. I see your point about the usage of `styled`, to be honest I didnt give much thought to that.:) I was mainly trying to separate things. I will convert it to CSS at some later point I guess. However some elements receive props and have some of its css applied based on different conditions, so I guess the `styled` element approach is appropriate there. What do you think? You are the css expert :)

**Comment by norbertdragan** [comment]
@piggydoughnut Yes, you will lose the ability to use props, so there isn't much point in converting to CSS at this stage.

**Pull request merged**